clio-linkchecking/src/main/resources/application.properties.example

```diff
@@ -1,3 +1,11 @@
+#Spring
+#logging.config=/data/logging/log4j2.xml
+logging.config=log4j2.xml
+spring.autoconfigure.exclude=\
+  org.springframework.boot.autoconfigure.mongo.embedded.EmbeddedMongoAutoConfiguration, \
+  org.springframework.boot.autoconfigure.mongo.MongoAutoConfiguration, \
+  org.springframework.boot.autoconfigure.data.mongo.MongoDataAutoConfiguration
+
```

Comment:
is this needed also? ```org.springframework.boot.autoconfigure.solr.SolrAutoConfiguration```

Reply:
I don't think so, but I will add it just in case for now.